### PR TITLE
refactor: fix flaky test failing due to local dev machine and CI server system timezone differences

### PR DESCRIPTION
**Pull request by Herraj**
## Goal
- We have this one test that is very flaky. Usually we've been getting past it by re-running the build at different times but it has been a hassle. Making it consistent across time zones now.
- fix other lint errors. 😕 These were caught by husky and I'm surprised weren't caught by the CI lint check??

### CODE REVIEW

File: src/curated-corpus/components/ScheduleItemFormConnector/ScheduleItemFormConnector.test.tsx [hunk 1]
```diff
@@ -172,9 +172,9 @@ describe('ScheduleItemFormConnector', () => {
     userEvent.click(datePicker);
 
     // Find today's date on the monthly calendar (it's actually a button)
-    const dates = screen.getAllByRole('button', {
-      name: today.toFormat('d'),
-    });
+    // const dates = screen.getAllByRole('button', {
+    //   name: today.toFormat('d'),
+    // });
```
Comment (Herraj) [inline src/curated-corpus/components/ScheduleItemFormConnector/ScheduleItemFormConnector.test.tsx:177]: This file has the unrelated changes to fix the lint check

File: src/curated-corpus/components/AddProspectModal/AddProspectModal.test.tsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ describe('The AddProspectModal component', () => {
             setIsRecommendation={setIsRecommendation}
             toggleApprovedItemModal={toggleApprovedItemModal}
             toggleModal={toggleModal}
-            toggleScheduleItemModal={toggleScheduleItemModal}
+            toggleDuplicateProspectModal={toggleDuplicateProspectModal}
```
Comment (Herraj) [inline src/curated-corpus/components/AddProspectModal/AddProspectModal.test.tsx:28]: surprised this wasn't caught by the CI lint check

**Review by jpetto — APPROVED** [review]
nice fix on the date-based test!
